communications/newsletters/newsletter_02_Oct2019.md

```diff
@@ -0,0 +1,92 @@
+# The Turing Way Newsletter: 04 October 2019
+
+Hello Turing Way friends!
+
+This month we had virtual coffees at **4 Online Collaboration Cafes** ([videos](https://www.youtube.com/channel/UCPDxZv5BMzAw0mPobCbMNuA), [more info](https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/online-collaboration-cafe.md)) :coffee: and discussed with you how you can contribute to this project.
+
+_The Turing Way_ was highlighted in the [The Turing impact story](https://www.turing.ac.uk/research/impact-stories/transforming-culture-data-science) authored by [Sean O'Neill](https://www.turing.ac.uk/people/business-team/sean-oneill) under the title **Transforming the culture of data science**.
+
+Shout out to Sarah Gibson and Rachael Ainsworth who gave talks and ran a workshop/demo-sesson on _The Turing Way_ Project. 
+Thanks to Paula Andrea Martinez and Philip Durbin for engaging with the project from Australia and USA 🌎 🌏 🌍 
+
+Say hello to [Malvika Sharan](https://twitter.com/MalvikaSharan), who joined _The Turing Way_ team as a Community Manager (and is drafting this newsletter :blush:).
+
+We are excited to meet many of you in London at [MozFest 2019](https://www.mozillafestival.org/en/). 
+
+That's all from us before Halloween! :jack_o_lantern: :maple_leaf:
+
+Find more details on these topics below. :point_down:
+
+## Community meetings
+
+### Next Collaboration Cafes: online call series
+
+This month’s collaboration cafes are scheduled on 2 October at 15:00 BST ([your local time](https://arewemeetingyet.com/London/2019-10-02/15:00/TuringWay-CollaborationCafe)), 16 October at 19:00 BST ([your local time](https://arewemeetingyet.com/London/2019-10-16/19:00/TuringWay-CollaborationCafe)), and 30 October at 08:00 GMT ([your local time](https://arewemeetingyet.com/London/2019-10-30/08:00/TuringWay-CollaborationCafe)).
```

Comment:
Do we need to rephrase this now that the first one has passed? Or is it ok? I'm in two minds!

Update: I think it make sense to be clear that one has passed....

```suggestion
We have three collaboration cafes this month!
The first was on 2 October at 15:00 BST and the next two are on 16 October at 19:00 BST ([your local time](https://arewemeetingyet.com/London/2019-10-16/19:00/TuringWay-CollaborationCafe)), and 30 October at 08:00 GMT ([your local time](https://arewemeetingyet.com/London/2019-10-30/08:00/TuringWay-CollaborationCafe)).
```